docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -132,6 +132,9 @@ property key | Description | Default |
 `plugin.tomcat.collect_http_params`| This config item controls that whether the Tomcat plugin should collect the parameters of the request. Also, activate implicitly in the profiled trace. | `false` |
 `plugin.springmvc.collect_http_params`| This config item controls that whether the SpringMVC plugin should collect the parameters of the request, when your Spring application is based on Tomcat, consider only setting either `plugin.tomcat.collect_http_params` or `plugin.springmvc.collect_http_params`. Also, activate implicitly in the profiled trace. | `false` |
 `plugin.http.http_params_length_threshold`| When `COLLECT_HTTP_PARAMS` is enabled, how many characters to keep and send to the OAP backend, use negative values to keep and send the complete parameters, NB. this config item is added for the sake of performance.  | `1024` |
+`plugin.springmvc.collect_http_headers`| This config item controls that whether the SpringMVC plugin should collect the headers of the request, activate implicitly in the profiled trace. | `false` |
```

Comment:
This is not necessary. header collecting should work once the `include_http_headers` not empty.

Reply:
```suggestion
```

Reply:
that's ok for me